src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -114,7 +114,7 @@ class Microsoft::Terminal::Core::Terminal final :
 #pragma region ITerminalInput
     // These methods are defined in Terminal.cpp
     bool SendKeyEvent(const WORD vkey, const WORD scanCode, const Microsoft::Terminal::Core::ControlKeyStates states, const bool keyDown) override;
-    bool SendMouseEvent(const COORD viewportPos, const unsigned int uiButton, const ControlKeyStates states, const short wheelDelta) override;
+    bool SendMouseEvent(const COORD viewportPos, const unsigned int uiButton, const ControlKeyStates states, const short wheelDelta, const Microsoft::Console::VirtualTerminal::TerminalInput::MouseButtonState state = {}) override;

```

Comment:
Are there any callers of this method that are actually relying on the default value of this param?

Reply:
Yeah. A MouseWheel event in `TermControl::_DoMouseWheel()`:
```c++
            return _terminal->SendMouseEvent(_GetTerminalPosition(point),
                                             WM_MOUSEWHEEL,
                                             _GetPressedModifierKeys(),
                                             ::base::saturated_cast<short>(delta));
```
The state would be ignored so there's no need to define it.

Reply:
i don't love default params. is there a downside to saying "there are no buttons pressed"?

what if you are scrolling _with buttons pressed_? then what? :P

Reply:
what i mean to say is, "is there a downside to making `_DoMouseWheel` explicitly say "there are no buttons pressed" instead of having a default parameter for a single caller"

Reply:
I love default params :P, but fair enough. Removing!